README.md

```diff
@@ -35,6 +35,9 @@ You will need the following things properly installed on your computer.
 
 * [Git][git]
 * [Node.js][node]
+  * v10–v15
+  * **Apple Silicon Users**:  if you experience problems building or running
+    Boundary UIs, try Node v15.
```

Comment:
Nice! The funny thing about this is the issue just reproduce when trying to build the ui from boundary project with `make build-ui` otherwise, to work directly on boundary-ui node v14 works better than v15. Since working with node v15 in the boundary-ui project I am still finding some small issues that I can't reproduce with v14.